Mount point fix using /etc/localtime file creation and pointing to it

ChaiBapchya opened the pull request:
Fixes the issue #44  of Mount denied - Path /etc/localtime is not shared from OS X and is not known to Docker.

Similar to docker/for-mac#2396
Solution that worked for me
```
$ mkdir ~/etc
$ vi ~/etc/timezone
```
Enter the timezone (mine was `America/New_York`)

Later updated the `docker-compose.yml` file with the following
```
  - ~/etc/timezone:/etc/localtime:ro
     # - /etc/localtime:/etc/localtime:ro #comment this
```

Hence this pull request.

ChaiBapchya committed:
Fixes #44

Fixes the issue of Mount denied - Path /etc/localtime is not shared from OS X and is not known to Docker.

One needs to create file /etc/timezone and add `America/New_york`

Asone commented:
Hi @ChaiBapchya, i believe this PR is on the right path, however adding a way to automate the file creation could make it a bit more complete. 

My current guess is that we can use bash to achieve that : 

`sudo mkdir -p ~/etc && touch ~/etc/localtime | /usr/sbin/systemsetup -gettimezone | sed 's/^Time Zone\: //' > ~/etc/localtime`

- It builds the `~/etc/localtime` file path
- `systemsetup` allows us to retrieve the used timezone
- `sed` cleans the unnecessary text
- `> ~/etc/localtime` to store timezone in the file

Adding it in a dockefile could make it

ChaiBapchya committed:
Automate /etc/localtime fix

Added command to automate process of creating /etc/localtime file and adding the Time Zone information

ChaiBapchya committed:
Automate /etc/localtime fix

Added command to automate process of creating /etc/localtime file and adding the Time Zone information

ChaiBapchya commented:
@Asone does this pull request work now?
